extensions/tiled-assets/README.md

```diff
@@ -0,0 +1,138 @@
+
+| Field Name    | Type    | Description                                                                                                        |
+| ------------- | ------- | ------------------------------------------------------------------------------------------------------------------ |
+| top           | number  | The size of the pixel-buffer in the top border of the image. Default is `0`.                                       |
+| left          | number  | The size of the pixel-buffer in the left border of the image. Default is `0`.                                      |
+| bottom        | number  | The size of the pixel-buffer in the bottom border of the image. Default is `0`.                                    |
+| right         | number  | The size of the pixel-buffer in the right border of the image. Default is `0`.                                     |
+| border_top    | boolean | Whether or not the pixel-buffer is included images on the top border of the first tile row. Default is `true`.     |
+| border_left   | boolean | Whether or not the pixel-buffer is included images on the left border of the first tile column. Default is `true`. |
+| border_bottom | boolean | Whether or not the pixel-buffer is included images on the bottom border of the last tile row. Default is `true`.   |
+| border_right  | boolean | Whether or not the pixel-buffer is included images on the right border of the last tile column. Default is `true`. |
+
+## Item fields
+
+| Field Name          | Type                                                         | Description                                                        |
+| ------------------- | ------------------------------------------------------------ | ------------------------------------------------------------------ |
+| asset_templates     | Map<string, [Asset Template Object](#asset-template-object)> | An array of Asset Template objects that denote templated arguments |
+
+### Asset Template Object
+
+An asset template is an object that contains a link template to data associated with the Item that can be downloaded or streamed. It is allowed to add additional fields.
+
+| Field Name    | Type   | Description                                                                           |
+| ------------- | ------ | ------------------------------------------------------------------------------------- |
+| href_template | string | **REQUIRED.** Link template to the asset object. Relative and absolute links are both allowed. All parts of the template within curly braces `{}` are meant to be template substitution parameters to be replaced |
+| title         | string | The displayed title for clients and users.                                            |
+| type          | string | [Media type](../README.md#media-types) of the asset.                                  |
```

Comment:
This link is broken.

Reply:
I fixed the link.

I see your point. Regarding directly linking the `Asset`: we would need put the template into the `href`, as it is mandatory, but I don't see the problem in that, to be honest.

Reply:
Nevermind, the issue I saw yesterday based on a wrong assumption. So linking directly to asset should be the best option, I think.